07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -0,0 +1,43 @@
+.nasa{
+  background-image: url("../img/nasa-background.jpg");
+  background-size: cover;
+  background-position: bottom;
+  background-repeat; no-repeat;
+  width: 850px;
+  height: 450px;
+  margin: 0 auto;
+  text-align: center;
+  margin-bottom: 10px;
+}
+h1{
+  padding-top: 180px;
+  font-weight: bold;
+  color: white;
+  font-size: 72px
+}
+h3{
+  padding-left:20px;
+}
+#info{
+  background-color: lightgray;
+  width: 850px;
+  height: 324px;
+  margin: auto;
+}
+ul{
```

Comment:
If you target the "li" and give them "padding: 0px;" - it should make them left align with the h1.